interpreter/script/js.ml

```diff
@@ -347,6 +346,8 @@ let wrap item_name wrap_action wrap_assertion at =
       {module_name = Utf8.decode "spectest"; item_name = Utf8.decode "eq_ref";
        idesc = FuncImport (4l @@ at) @@ at} @@ at ]
   in
+  let idesc_is_func = match idesc.it with FuncImport _ -> 1l | _ -> 0l in
```

Comment:
I don't completely understand the logic here, but for me it causes an off-by-one error. Just using
`let item = Lib.List32.length imports @@ at in` in the next line fixes the problem for me.

Reply:
Oops, that went the wrong way round. Replaced with a correct and less brittle version.